charts/kubescape-operator/tests/snapshot_test.yaml

```diff
@@ -1,4 +1,20 @@
 suite: Snapshot tests
+kubernetesProvider:
+  scheme:
+    "v1/Secret":
+      gvr:
+        version: v1
+        resource: secrets
+      namespaced: true
+  objects:
+    - kind: Secret
+      apiVersion: v1
+      metadata:
+        name: extra-certificates
+        namespace: kubescape
+      data:
+        cert1: Zm9v
```

Comment:
@slashben it doesn't seem to work...

Reply:
See this https://github.com/kubescape/helm-charts/blob/5013f681dcb122f88f7668893a1f1158b96768cc/charts/kubescape-operator/templates/node-agent/daemonset.yaml#L251

it looks like `lookup` is not easily testable. I don't see a way to mock it, on the other hand I don't know how to accommodate the requirement. 

This might not be covered by the unittest, but it works on a real cluster